interactive/widget_style.css

```diff
@@ -0,0 +1,17 @@
+.stButton > button {
+    background: linear-gradient(45deg, #09B3BF 0%, #5FE3C2 100%);
+    max-width: 250px;
+    align-items: end;
+}
+.element-container:nth-child(0) {
+    text-align: center;
+}
+.css-1kyxreq {
```

Comment:
What are those class names? If it is created dynamically by streamlit it can lead to multiple errors as it is mechanism for name mangling

Reply:
It's a consistent reference to a single class containing an image - it remains constant in all of the instances: Docker local, Docker cloud, and web implementation.